packages/@okta/vuepress-theme-prose/global-components/HomePageBanner.vue

```diff
@@ -0,0 +1,71 @@
+<template>
+  <div class="home-page-banner">
+    <div :class="{ 'home-page-banner--heading': true, 'home-page-banner--heading-hidden': !$slots.heading }">
+      <slot name="heading" />
+    </div>
+    <div class="home-page-banner--description">
+      <slot name="description" />
+    </div>
+
+    <div
+      v-if="primaryCtaText || secondaryCtaText"
+      class="home-page-banner--cta-container"
+    >
+      <a
+        v-if="primaryCtaLink"  
```

Comment:
Let's also check here for `primaryCtaText`. If we remove link and leave text that will break UI
<img width="706" alt="image" src="https://github.com/okta/okta-developer-docs/assets/4158731/78b7dd83-ae89-4560-973a-8c56412e5df6">

Reply:
Nice catch.. done!